### PR TITLE
Firebase initial setup

### PR DESCRIPTION
**Pull request by LadyKerr**
# Description

Describe change or new feature here.

## Checklist

Remove any items which are not applicable.

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+import rootReducer from "./reducers/rootReducer";
+import { reduxFirestore, getFirestore } from "redux-firestore";
+import { reactReduxFirebase, getFirebase } from "react-redux-firebase";
+import fbConfig from "./config/fbConfig";
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const store = createStore(
+  rootReducer,
+  composeEnhancers(
```
Comment (micahjones13) [inline src/index.js]: We might run into a problem when compiling with this and also using the redux dev tools extension. I ran into an error trying to do both before, so just a heads up

Comment (micahjones13) [reply src/index.js]: To clarify - the composeEnhancers is what i'm referring to

Comment (LadyKerr) [reply src/index.js]: I see, how would you rec we resolve?

**Comment by Full-lifey** [comment]
Merging requests into staging. Debugging completed in following PR https://github.com/labs-future-hope/Future-Hope-Client/pull/9

**Pull request merged**